internal/proto/codec/error_holder_codec.go

```diff
@@ -24,12 +24,6 @@ const (
 	ErrorHolderCodecErrorCodeInitialFrameSize = ErrorHolderCodecErrorCodeFieldOffset + proto.IntSizeInBytes
 )
 
-/*
-type errorholderCodec struct {}
-
-var ErrorHolderCodec errorholderCodec
-*/
-
```

Comment:
🤔 are these comments coming from client protocol generator? if we have this comment in the client protocol generator, and the comment is not useful, we can remove it from there as well.

Reply:
Yes, they are. I'm working on updating the client protocol generator in another branch.